contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -1,6 +1,6 @@
 ---
-date: 2022-12-08
-title: "Product engineer vs software engineer: what's the difference?"
+date: 2023-05-10
+title: "Product engineers vs software engineers: How are they different?"
```

Comment:
```suggestion
title: "Product engineer vs software engineer: How are they different?"
```
The keyword that gets the most traffic is "Product engineer vs software engineer" so I think we should keep it singular.